durst/src/main.rs

```diff
@@ -18,13 +18,12 @@ use std::rc::Rc;
 use std::sync::Mutex;
 use std::time::Duration;
 
-use config::Rule;
 use notification::Notification;
 
 #[derive(Debug)]
 struct Container {
     queue: Vec<Notification>,
-    config: Vec<Rule>,
+    config: config::DurstConfiguration,
```

Comment:
minor: would prefer to have a `use config::DurstConfiguration;` at the top, so this simply says `DurstConfiguration`